### PR TITLE
Feature/300 refactor resume profile

### PR DESCRIPTION
**Pull request by alex-flores-jalafund**
[AB#300](https://dev.azure.com/funjaladev/DevLevel3/_boards/board/t/DevLevel3%20Team/Stories/?workitem=300)

Refactor desing resume page
follow the next detail

image

### CODE REVIEW

File: src/components/resumes/Details.css [hunk 1]
```diff
@@ -1,14 +1,92 @@
...
+  display: flex;
+}
+.head-name-address {
+  margin: 0px;
+  padding: 0px;
+}
+.head-detail-name {
+  padding: 20px 20px 10px 0px;
+}
+.prueba {
```
Comment (rodrigo-baldiviezo-jalafund) [inline src/components/resumes/Details.css]: change word to english word maybe?

File: src/components/resumes/Details.jsx [hunk 2]
```diff
@@ -25,22 +30,59 @@ const useStyles = makeStyles((theme) => ({
...
   paper: {
     color: theme.palette.text.secondary,
+    margin: "20px 0px",
     padding: theme.spacing(4),
+    width: "1000px",
   },
-  root: {
-    "& .MuiTextField-root": {
-      margin: theme.spacing(2),
+  prueba: {
```
Comment (rodrigo-baldiviezo-jalafund) [inline src/components/resumes/Details.jsx]: same here

File: src/components/resumes/Details.jsx [hunk 3]
```diff
@@ -65,7 +107,7 @@ function Details() {
     setOpenModal(true);
   };
 
-  useEffect(() => {
+  fetch(() => {
```
Comment (rodrigo-baldiviezo-jalafund) [inline src/components/resumes/Details.jsx]: I not sure about this change, can you explain me please?

File: src/components/resumes/Details.css (hunk 1 above)
Comment (alex-flores-jalafund) [reply src/components/resumes/Details.css]: It's done

File: src/components/resumes/Details.jsx (hunk 2 above)
Comment (alex-flores-jalafund) [reply src/components/resumes/Details.jsx]: It's done

File: src/components/resumes/Details.jsx (hunk 3 above)
Comment (alex-flores-jalafund) [reply src/components/resumes/Details.jsx]: it's a error I have changed to useEffect

**Review by rodrigo-baldiviezo-jalafund — APPROVED** [review]
LGTM

**Review by alejandro-castellon — APPROVED** [review]
LGTM

**Review by jorge-lema-jalafund — APPROVED** [review]
approved

**Pull request merged**